content/documentation/pages/5-feature-guides/2-streams/12-monitoring.md

```diff
@@ -75,6 +87,20 @@ You should see dashboards similar to those shown in the following image:
 
 ![SCDF Grafana Prometheus](images/grafana-prometheus-scdf-applications-dashboard.png)
 
+<!--Wavefront -->
+
+### Wavefront
+
+To install Data Flow with Wavefront support, follow the [Monitoring with Wavefront](%currentPath%/installation/local/docker-customize/#wavefront) Docker Compose instructions. This will bring up Spring Cloud Data Flow, Skipper, Apache Kafka, and point it to the Wavefront Data Flow Integration tile.
```

Comment:
```suggestion
To install Data Flow with Wavefront support, follow the [Monitoring with Wavefront](%currentPath%/installation/local/docker-customize/#wavefront) Docker Compose instructions. This will bring up Spring Cloud Data Flow, Skipper, Apache Kafka, and it will also point to the Wavefront's Data Flow Integration Tile automatically.
```